### DISCUSSION TITLE
Import pre-trained nnUNet

### DISCUSSION THREAD

**Discussion by aksg87** [Project-MONAI/MONAI#2871] [Q&A]
Hello,

Is it possible to import a pre-trained `nnU-Net` using the `DynUNet` architecture? If there are any references on this workflow, I would love to see them! Thank you in advance.

**Comment by yiheng-wang-nv** [comment]
Hi @aksg87 , if all hyper-parameters are the same, it should be possible. However, the names of some layers may be different, thus you need to map the names. For example: https://github.com/Project-MONAI/MONAI/blob/dev/monai/networks/nets/senet.py#L272
